incubator/hnc/internal/validators/anchor.go

```diff
@@ -79,8 +79,13 @@ func (v *Anchor) handle(req *anchorRequest) admission.Response {
 	switch req.op {
 	case k8sadm.Create:
 		// Can't create subnamespaces in excluded namespaces
-		if config.EX[pnm] {
-			msg := fmt.Sprintf("The namespace %s is not allowed to create subnamespaces. Please create subnamespaces in a different namespace.", pnm)
+		if config.ExcludedNamespaces[pnm] {
+			msg := fmt.Sprintf("The namespace %q is not allowed to create subnamespaces. Please create subnamespaces in a different namespace.", pnm)
+			return deny(metav1.StatusReasonForbidden, msg)
+		}
+		// Can't create subnamespaces using excluded namespace names
+		if config.ExcludedNamespaces[cnm] {
+			msg := fmt.Sprintf("%q is not allowed to be created as a subnamespaces. Please use a different name for the subnamespace.", cnm)
```

Comment:
Maybe: "The namespace %q is excluded from HNC and cannot be created as a subnamespace."

Reply:
Updated to:
"Cannot create a subnamespace in the excluded namespace %q"
"Cannot create a subnamespace using the excluded namespace name %q",
"Cannot create a subnamespace using an existing namespace name %q"